scripts/options.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(() => {
+  // Saves options to chrome.storage.sync.
+  const apiKey = $('#api_key');
+
+  function getFormData() {
+    const formElements = $('form').children('input:not(#save)');
+
+    const formData = {};
+
+    formElements.each(() => {
+      formData[this.name] = this.value;
+    });
+
+    return formData;
+  }
+
+  function saveOptions() {
+    const saveValues = getFormData();
+
+    saveValues[apiKey.attr('name')] = apiKey.val();
```

Comment:
No you are right, but the function was not working correctly. Fixed now, good catch.